crates/rolldown_binding/src/types/binding_normalized_options.rs

```diff
@@ -275,4 +275,9 @@ impl BindingNormalizedOptions {
   pub fn minify_internal_exports(&self) -> bool {
     self.inner.minify_internal_exports
   }
+
+  #[napi(getter)]
+  pub fn context(&self) -> String {
```

Comment:
We had a test for this, could add this field in that test too?

https://github.com/rolldown/rolldown/blob/cf09f465b87e0d4b6cf1071ad132314f3b56b13c/packages/rolldown/tests/fixtures/verify-options/_config.ts